.goreleaser.yml

```diff
@@ -53,6 +53,12 @@ changelog:
       - "^infra:"
       - "^build\\(deps\\):"
       - "^Merge pull request"
+  groups:
+    - title: Breaking Changes
+      regexp: "^.*!:+.*$"
```

Comment:
The convention is the with the exclamation point

Reply:
I believe this is [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), correct?

Reply:
>BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change

A link to the CC site could be helpful 🙂

Reply:
There's a footnote to it in the contributing guidelines.

Reply:
Thanks, don't know how I missed that. Actually, I think I came from the changelog link to the commit so my context was the commit, rather than the pull request that contained it. That's good to know.